charactersheet/charactersheet/templates/armors.tmpl.html

```diff
@@ -198,6 +199,14 @@
                         class="btn btn-primary"
                         data-bind='click: addArmor'
                         data-dismiss="modal">Add</button>
+                <p class="text-muted text-left" data-bind='visible: shouldShowDisclaimer'>

```

Comment:
* The SRD label never goes away. Look at the items and spells modal to see how we toggle the label.